python/requirements_compiled.txt

```diff
@@ -143,7 +143,7 @@ autorom==0.6.1 ; platform_machine != "arm64"
     # via -r /ray/ci/../python/requirements/ml/rllib-test-requirements.txt
 autorom-accept-rom-license==0.6.1
     # via autorom
-aws-sam-translator==1.78.0
+aws-sam-translator==1.79.0
```

Comment:
I feel that this compiled file should be different for different python version in general, because different python version will generate different results when resolving dependencies.

this is already causing a super long python deps installation on some builds.

this particular change is probably safe though.

Reply:
agreed, also because dependencies have some releases that are only supported on specific python versions

it has been a big headache trying to get a compiled version that works across the versions here.